.github/workflows/production-deploy.yml

```
@@ -0,0 +1,68 @@
+      - name: Run Fastlane (Build & Distribute)
+        env:
+          SUPPLY_JSON_KEY: ${{ github.workspace }}/app/google-play-api.json
+          KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
+          KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
+          KEYSTORE_PASSWORD: ${{ secrets.KEYSTORE_PASSWORD }}
+          KEYSTORE_FILE_PATH: ${{ github.workspace }}/app/release.jks
+        run: fastlane android distribute
+
+      # --- GITHUB RELEASE (For FOSS/IzzyOnDroid) ---
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)